jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -156,6 +156,42 @@ pipeline {
                             } else {
                                 echo "Skipping publishing snapshots, ${mavenPath} does not exist."
                             }
+                        }
+                    }
+                    post {
+                        always {
+                            postCleanup()
+                        }
+                    }
+                }  
+                stage('build-os-snapshot-linux-x64-tar') {
```

Comment:
Can you say `build-OpenSearch-Only-snapshot-linux-x64-tar` Its easy to read and understandable.

Reply:
I think all lower case is better here.
Be careful touching these stage names as the notification might retrieve msg based on the stage names.

Reply:
Rename done:
build-bundle-snapshot-linux-x64-tar
build-opensearch-snapshot-linux-x64-tar
build-opensearch-snapshot-linux-arm64-tar
build-opensearch-snapshot-macos-x64-tar
build-opensearch-snapshot-windows-x64-zip